towhee/hub/builtin/operators/computer_vision.py

```diff
@@ -227,22 +227,34 @@ class save_image:
     >>> (
     ...     towhee.dc['path'](['https://github.com/towhee-io/towhee/raw/main/towhee_logo.png'])
     ...           .image_decode['path', 'img']()
-    ...           .save_image['img','path_new'](dir='temp/pic')
+    ...           .save_image['img','path_new'](dir='temp/pic', format="jpg", keep_original_filename = False)
     ...           .to_list()
     ... )
     [<Entity dict_keys(['path', 'img', 'path_new'])>]
     """
 
-    def __init__(self, dir: str):
+    def __init__(self, dir: str, format: str, keep_original_filename: bool):
```

Comment:
I think the `keep_original_filename` parameter is unnecessary, and `format` can defaults to '.jpg', like:`def __init__(self, dir: str, format: str='.jpg'):`